src/shared/logToSentry.ts

```diff
@@ -0,0 +1,26 @@
+import { z } from 'zod';
+// other details like timestamp, and hostname/ IP address should be included by Sentry
+const zLogObject = z.object({
+    message: z.string(),
+    email: z.string().email().optional(),
+    data: z.object({}).optional()
+})
```

Comment:
Add semi-colon. Also, in a different PR, can you help enable eslint check for semi-colons?